patch/net-psample-module-unload.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/net/psample/psample.c b/net/psample/psample.c
```

Comment:
can you provide the description in this patch file?

Reply:
updated